src/python/grpcio/grpc/_adapter/_c/types/channel.c

```diff
@@ -164,7 +164,7 @@ PyObject *pygrpc_Channel_watch_connectivity_state(
   int last_observed_state;
   CompletionQueue *completion_queue;
   char *keywords[] = {"last_observed_state", "deadline",
-                      "completion_queue", "tag"};
+                      "completion_queue", "tag", NULL};
```

Comment:
I thought the format string determined the length of the keywords array. Why is this being added?